charts/loki-distributed/templates/_helpers.tpl

```diff
@@ -141,6 +141,17 @@ Return the appropriate apiVersion for PodDisruptionBudget.
   {{- end -}}
 {{- end -}}
 
+{{/*
```

Comment:
Pretty please, add in support to customize the `behavior` on the autoscalers too. We find that we cannot slow down the scale up/down behavior and that causes problems for us.

1. We can't create our own HPAs because as soon as you set `ingester.autoscaling.enabled: false`, then `replicas: ...` is set explicitly.
2. We can't pass in the `behavior` settings we want to slow down the autoscaling.